tools/cldr-code/src/main/java/org/unicode/cldr/util/Emoji.java

```diff
@@ -337,17 +323,12 @@ public static UnicodeSet getNonConstructed() {
     }
 
     private static Set<String> NAME_PATHS = null;
-    private static Set<String> KEYWORD_PATHS = null;
     public static final String TYPE_TTS = "[@type=\"tts\"]";
 
     public static synchronized Set<String> getNamePaths() {
         return NAME_PATHS != null ? NAME_PATHS : (NAME_PATHS = buildPaths(TYPE_TTS));
     }
 
-    public static synchronized Set<String> getKeywordPaths() {
```

Comment:
ditto

Reply:
I also just now searched for getKeywordPaths in unicodetools and it wasn't found -- but we should have some way of encapsulating such dependencies!

Reply:
I had never built unicodetools before. Just now I downloaded it, ran `mkdir -p Generated/BIN`, and ran `mvn install package -DCLDR_DIR=/[...]/cldr`. All the tests passed. There were warnings but no indication of methods missing from cldr.

Reply:
ok, good